test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -951,6 +951,30 @@ class test_ucp_sockaddr_protocols : public test_ucp_sockaddr {
         ASSERT_UCS_OK(status);
     }
 
+    void test_am_send_recv(size_t size)
+    {
+        std::string sb(size, 'x');
+
+        bool am_received = false;
+        ucp_worker_set_am_handler(receiver().worker(), 0,
```

Comment:
where we remove it?

Reply:
what API do you mean?

Reply:
ucp_worker_set_am_handler
```
 * @param [in]  cb          Active Message callback. NULL to clear.
```

Reply:
is it required by API? What is the reason to clear in this test?

Reply:
more for a good practice to release what is allocated